CHANGELOG.md

```diff
@@ -26,7 +26,10 @@ This section is for maintaining a changelog for all breaking changes for the cli
 - Delete shape property ([#884](https://github.com/opensearch-project/opensearch-java/pull/885))
 
 ### Fixed
-- Fix version and build ([#254](https://github.com/opensearch-project/opensearch-java/pull/254))
+- Fix version and build ([#254](https://github.com/opensearch-project/opensearcjava/pull/254))
```

Comment:
There's a typo in opensearch-java here, so the link checker caught it and that's why the build failed.